backend/typescript/migrations/2024.06.12T01.36.05.create-behaviour-table.ts

```diff
@@ -0,0 +1,24 @@
+import { DataType } from "sequelize-typescript";
```

Comment:
isn't this daniel's migration file? try rebasing, this file shouldn't be in your pr